### DISCUSSION TITLE
deepin20.6 三个使用痒点反馈

### DISCUSSION THREAD

**Discussion by doooooit** [linuxdeepin/developer-center#3295] [Features Request & Ideas | 特性请求 & 头脑风暴]
1. dock 栏“智能隐藏”模式，当应用窗口最大化时可以隐藏，鼠标移到接近屏幕下缘（不用挨到下缘，只要接近就触发）显示。

    但这个逻辑有个严重问题，很多应用——尤其是一些文本编辑类的应用——其应用窗口底部状态栏有可操作区域，这些应用在窗口最大化情况下永远点击不到状态栏，只要鼠标移到状态栏的位置就会触发弹出 dock 栏遮挡应用本身

    这一点参考 macOS 的做法，应用全屏状态时也会隐藏 dock 栏，也是鼠标移动到屏幕下缘显示，但只有在鼠标指针真的挨到最底部才会触发，完全不会误触。

    **建议：“智能隐藏”模式 dock 栏只有在鼠标完全接触到屏幕最底部时才触发显示**

2. 多桌面模式，使用快捷键进行桌面间切换时是循环模式，在最右边的桌面再往右切换会回到最左边的桌面。

    但是这样逻辑会造成误导，使用多个桌面时桌面之间的关系就是依靠左右来区分和记忆的，最右边的桌面还能往右切换会打乱用户记忆，不记得切换过来的这个桌面是哪一个桌面

    **建议：多桌面不要循环切换**

3. deepin 已经很完善，各个系统功能和设计我愿意评价为优秀。但启动器是我唯一认为不及格的部分，有多个槽点

    1. 打开/关闭 启动器完全无动画，生硬无比
    2. 启动器以前是上下滚动的，现在变成了左右分页，其实并不觉得左右分页比上下滚动更好
    3. 左右分页也就罢了，但现在每次打开启动器都会回到第一页，无论我上次关闭时是在哪一页
    4. 左右分页切换时动画生硬、掉帧
    5. 启动器应用图标大小不统一，鼠标悬浮时的半透明蒙版也是大小不统一，而且不是正方形，强迫症难受
    6. 无法创建文件夹，无法归类多个应用

    **建议：参考 macOS 的启动器**

**Comment by BLumia** [comment]
- 智能隐藏误触发的问题我记得有个近似的反馈，但一时半会儿没找到在哪...
- 多桌面循环这个我觉得是见仁见智的事情，不同人用法不太一样。
- 目前我在 Arch Linux 下使用 deepin-wm + DDE，启动器动画我这边目前是有个淡入淡出的，但我的另一个 deepin v20.6 的环境确实没有动画。说不定是窗管的事。
- 创建文件夹来归类应用这个是有计划的，但得等 v23 才会有，[之前也有相应的反馈](https://github.com/orgs/linuxdeepin/discussions/3281)。

其它请求我觉得还是挺中肯的， @meiyixiang 可以看下 :)

**Comment by meiyixiang** [comment]
1.属于交互体验性问题，在V20阶段不会去做修改，因为V23会提供全新的桌面设计和交互
2.问题2我已经记录会提交给产品，希望在新桌面设计时会考虑到此问题。
3.这个问题明确待V23全新桌面发布时会支持对启动器应用分类管理功能。
